docs/CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog
 =========
 
+v0.1.5 (2021-11-25)
```

Comment:
this should be `new_version`. Nobody knows if the next version is really 0.1.5. Maybe there is release before that one. ;-)

```suggestion
new_version
```

Reply:
Ah, that makes sense, thanks!